bitnami/common/templates/validations/_mongodb.tpl

```diff
@@ -106,3 +110,16 @@ Params:
     {{- .context.Values.architecture -}}
   {{- end -}}
 {{- end -}}
+
+Usage:
+{{ include "common.mongodb.values.bypassUpgradeValidation" }}
+Params:
+  - subchart - Boolean - Optional. Whether to bypassUpgradeValidation for password environment variables. Default: false
+*/}}
+{{- define "common.mongodb.values.bypassUpgradeValidation" -}}
+  {{- if .subchart -}}
+    {{- .context.Values.mongodb.bypassUpgradeValidation -}}
+  {{- else -}}
+    {{- .context.Values.bypassUpgradeValidation -}}
+  {{- end -}}
+{{- end -}}
```

Comment:
Apart from using Vault to manage the credentials, under what other circumstances are you interested on bypassing the validations? If Vault is the unique case, then let's skip the validations when `auth.vaultCredentials` is set

Reply:
This is added to support the upgrade use case, will revisit it and check if the any other use case needs to be covered.

Reply:
Answered above.